buildSrc/subprojects/build/build.gradle.kts

```diff
@@ -1,5 +1,9 @@
 dependencies {
     compile("commons-lang:commons-lang:2.6")
     compile("com.google.guava:guava-jdk5:14.0.1")
-    compile("org.asciidoctor:asciidoctor-gradle-plugin:1.5.6")
+    compile("org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.11") {
+        exclude(module="asciidoctorj")
```

Comment:
I'm sure we can do better than this with the new dependency resolution features. What's the reason for this `exclude`?

Reply:
Sorry, copied this from one of the [asciidoctor-pdf issues](https://github.com/asciidoctor/asciidoctor-pdf/issues) (I forget which). Anyway, I don't think this is necessary anymore. The intent was to enforce a particular version of asciidoctorj run by all tools.